### DISCUSSION TITLE
Is there a way to output only the transcript and no timestamps?

### DISCUSSION THREAD

**Discussion by patrickmac110** [Purfview/whisper-standalone-win#112] [Q&A] [answered]
The timestamps are nice for a lot of use cases, sure, but there's also plenty of use cases (writing blog posts, etc.) that I want to just have the text and no time stamps at all.

**Comment by Purfview** [comment, marked as answer]
`-f=text`

**Reply by rajeevchourasiya** [reply]
can you please tell me how to use it, as I'm using it like this:  
![image](https://github.com/Purfview/whisper-standalone-win/assets/130376260/ec956ffa-8dc3-4151-940c-e52e3a489f34)
"fw.exe "C:\Users\rajee\Documents\Sound recordings\test.m4a" --task transcribe --language English --output_format -f=txt --output_dir source"  
But it wasn't working, the output looks like this: 
![image](https://github.com/Purfview/whisper-standalone-win/assets/130376260/f098772b-e6ad-46fd-acec-443228f1a241)

also it is repeating the last bit of the output, as can be seen in the output, please help me in resolving this. TIA.

**Reply by Purfview** [reply]
It's "text", not "txt", remove "--output_format" as it's dupe of "-f".
If you have other issue then open them in the new issue -> https://github.com/Purfview/whisper-standalone-win/issues/new
